src/components/InputName/InputName.tsx

```diff
@@ -74,18 +75,84 @@ const InputElement = styled.input`
     font-family: 'Futura Std', Arial, Helvetica, sans-serif;
   }
 `
+const Button = styled.button`
+  position: absolute;
+  border: none;
+  background: none;
+  cursor: pointer;
+  bottom: 13px;
+  right: 17px;
+  @media (min-width: 400px) {
+    right: 38px;
+  }
+`
+
+const LabelContainer = styled.div`
+  display: flex;
+`
 
 export interface InputProps {
   label: string
   placeholder: string
   name: string
+  type: string
+  validInput: boolean
+  inputValue: string
+  onChange: React.ChangeEventHandler<HTMLInputElement> | undefined
+  errorMessage?: string
 }
 
-export const Input: React.FC<InputProps> = ({ label, placeholder, name }) => {
+export const Input: React.FC<InputProps> = ({
+  label,
+  placeholder,
+  name,
+  type,
+  validInput,
+  inputValue,
+}) => {
+  const [showPassword, setShowpassword] = useState(false)
+
   return (
     <InputContainer>
-      <label className='labelStyle'>{label}</label>
-      <InputElement placeholder={placeholder} className={name} />
+      <div style={{ position: 'relative' }}>
+        {!validInput ? (
+          <LabelContainer>
+            <label className='labelStyle'>{label}</label>
+            <small
+              style={{
+                color: '#FC462B',
+                display: 'flex',
+                width: 'fit-content',
+                marginLeft: 'auto',
+                fontSize: '16px',
+                lineHeight: '19px',
+                marginBottom: '10px',
+                textTransform: 'capitalize',
+              }}>
+              {name} invalid
+            </small>
+          </LabelContainer>
+        ) : (
+          <label className='labelStyle'>{label}</label>
+        )}
+        <InputElement
+          style={{
```

Comment:
If we can avoid inline styles, let's stay away from it.

Reply:
Oh, it can be very hard to style the input without inline-styles in this case.